src/ts/Offers/index.tsx

```diff
@@ -10,37 +10,44 @@ import debug from 'debug';
 const log = debug('app:Offers');
 import { offers } from '../Detail/mockup';
 import { default as AppBarVG } from '../AppBar';
+const limit = 4;
```

Comment:
or even put that magic number into some conf file?
eventually, that conf file could be used for the cms